xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/changesdoc.vm

```diff
@@ -32,6 +58,12 @@
       #end
       </div> ## changes-info-content
     </div> ## changes-info-box-to
+    #if ("$!previousLink" != '')
+      <a href="$previousLink" id="arrow-left-shift" class="changes-arrow-left">&lt;</a>
+    #end
+    #if ("$!nextLink" != '')
+      <a href="$nextLink" id="arrow-right-shift" class="changes-arrow-right">&gt;</a>
```

Comment:
On the CSS part I don't have anything else to add. 
On the HTML, do you really need all those IDs and classes: #arrow-right-shift, #arrow-left-shift, .changes-arrow-left, .changes-arrow-right, .changes-arrow ?

Reply:
I needed the classes for the css. I've added the ids to be able to easily locate the elements for the functional tests. I know it's a bit uggly so I can try to remove the ids and locate the element using XPath (it's currently used for some tests with our version of Selenium I think)